pages/osx/dd.md

```diff
@@ -1,19 +0,0 @@
-# dd
```

Comment:
Yes, I recall that discussion (#555). Let's keep two pages then.

ps - this is the third time these small differences in dd raise confusion. A "notes" section (c.f. #481) would probably prevent this sort of common pitfalls.